code/modules/fabrication/fabricator_bioprinter.dm

```diff
@@ -9,28 +11,75 @@
 	base_storage_capacity = list(
 		/decl/material/solid/meat = SHEET_MATERIAL_AMOUNT * 100
 	)
-	var/datum/dna/loaded_dna_datum
+	var/datum/dna/loaded_dna //DNA for biological organs
+
+/obj/machinery/fabricator/bioprinter/get_nano_template()
+	return "fabricator_bioprinter.tmpl"
+
+/obj/machinery/fabricator/bioprinter/make_order(datum/fabricator_recipe/recipe, multiplier)
+	var/datum/fabricator_build_order/order = ..()
+	//Keep these in the order so changing settings while queueing things up won't screw up older orders in the queue
+	LAZYSET(order.data, "dna", loaded_dna)
```

Comment:
Can you explain what you mean here by wrapper?
Because orders are temporary objects and not really meant to be tempered with. And I'm not sure if its worth it creating datum subobjects for that list?

Reply:
Well, the core change here is that instead of passing around the recipe + multiplier + optional args, you are now passing around instances of `datum/fabricator_build_order`. My point is that it's better form to not manipulate this datum's internal structure from the outside, as that makes it easy to modify that internal structure in the future. If you really don't care about this and are using this purely to work around lack of multiple returns, then you can ignore this + the comment below about moving build logic.

Reply:
pushed the change!